src/pages/VenueEntrancePage/VenueEntrancePage.tsx

```diff
@@ -37,36 +36,47 @@ export const VenueEntrancePage: React.FC = () => {
   const { user, profile } = useUser();
   const { step: unparsedStep } = useParams<{ step?: string }>();
 
-  const { spaceSlug } = useSpaceParams();
-  const { space, spaceId, isLoaded: isSpaceLoaded } = useSpaceBySlug(spaceSlug);
+  const { worldSlug, spaceSlug } = useSpaceParams();
+  const { world, space, spaceId, isLoaded } = useWorldAndSpaceBySlug(
+    worldSlug,
+    spaceSlug
+  );
 
-  const { world, isLoaded: isWorldLoaded } = useWorldById(space?.worldId);
   const step = Number.parseInt(unparsedStep ?? "", 10);
 
   const proceed = useCallback(
-    () => spaceSlug && history.push(venueEntranceUrl(spaceSlug, step + 1)),
-    [spaceSlug, step, history]
+    () =>
+      spaceSlug &&
+      history.push(venueEntranceUrl(worldSlug, spaceSlug, step + 1)),
+    [worldSlug, spaceSlug, step, history]
   );
 
-  if (!isSpaceLoaded || !isWorldLoaded) {
+  if (!isLoaded) {
     return <LoadingPage />;
   }
 
-  if (!spaceId || !space || !spaceSlug) {
+  if (!spaceId || !space || !spaceSlug || !world) {
     return <NotFound />;
   }
 
-  const stepConfig = world?.entrance?.[step - 1];
+  const stepConfig = world.entrance?.[step - 1];
   if (!stepConfig) {
-    return <Redirect to={venueInsideUrl(spaceSlug)} />;
+    return (
+      <Redirect
+        to={generateAttendeeInsideSpaceUrl({
+          worldSlug: world.slug,
+          spaceSlug: spaceSlug,
```

Comment:
Should be the same, no?

```suggestion
          worldSlug,
          spaceSlug,
```